core/chainlink.Dockerfile

```diff
@@ -38,13 +38,14 @@ FROM ubuntu:20.04
 
 ARG CHAINLINK_USER=root
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y ca-certificates gnupg lsb-release curl
+RUN apt-get update && apt-get install -y ca-certificates gnupg lsb-release curl patchelf
```

Comment:
Should patchelf only be installed on the goreleaser built docker images?

Reply:
yeah good catch! the regular Dockerfile doesn't need this.